sprinter/lib/__init__.py

```diff
@@ -45,6 +46,12 @@ def prompt(prompt_string, default=None, secret=False, boolean=False, bool_type=N
     prompt_string += (default_msg.format(val=default) if default else ": ")
     if secret:
         val = getpass(prompt_string)
+    elif populated_default is not None:
+        readline.set_startup_hook(lambda: readline.insert_text(populated_default))
+        try:
+           val = raw_input(prompt_string)
+        finally:
+           readline.set_startup_hook()
```

Comment:
It would be nice to add colors to the prompts to make it more obvious that this is an editable value. Maybe using something like this: https://pypi.python.org/pypi/termcolor